webpack.config.js

```diff
@@ -36,49 +35,6 @@ let config = dev.getDefaultWebpackConfig({
     },
 
     loaderRules: [
-        {
```

Comment:
OMG, love it, no more webpack goo!! ☺️

Reply:
Gonna have to put some goo back; it's not copying resources from AzExt...